rpm/gen-rpm-ver

```diff
@@ -47,12 +48,12 @@ if [[ "$rpmVersion" == *-dev ]] || [ -n "$($GIT_COMMAND status --porcelain)" ];
     gitUnix="$($GIT_COMMAND log -1 --pretty='%at')"
     gitDate="$($DATE_COMMAND --date "@$gitUnix" +'%Y%m%d.%H%M%S')"
     gitCommit="$($GIT_COMMAND log -1 --pretty='%h')"
-    gitVersion="git${gitDate}.0.${gitCommit}"
     # gitVersion is now something like '20150128.112847.17e840a'
-    rpmVersion="${rpmVersion%-dev}"
-    rpmRelease="0.0.dev.$gitVersion"
+    rpmVersion="0.${gitDate}~${gitCommit}"
```

Comment:
i'm concerned about this [guideline](https://fedoraproject.org/wiki/Packaging:Versioning) from fedora:
> The tilde (~) notation which alters the way RPM does version comparisons MUST NOT be used.

Reply:
Not really it's that big of an issue:

```
❯ rpmdev-vercmp 1.0.0-~a 1.0.0~b
1.0.0-~a > 1.0.0~b
```

Related issue: https://bugzilla.redhat.com/show_bug.cgi?id=734802

`tilde` support was added in RPM 4.10.0 http://rpm.org/wiki/Releases/4.10.0

Reply:
Also the tilde isn't really being used as a **sortable** instrument, it's more just used to delineate between the timestamp and the commit sha.

Not really worried about the `tilde` affecting the `sortability` of the RPM packages.